File: CSharp/FactorsOfGivenNumber.cs
using System.Linq;

namespace CSharp
{
    // Create a function that finds each factor of a given number n.
    // Your solution should return an array of the number(s) that meet this criteria.
    // https://edabit.com/challenge/WR3Q72Jbv7Kyzu6AT
    public static class FactorsOfGivenNumber
    {
        public static int[] FindFactors(int num) => Enumerable.Range(1, num).Where(i => num % i == 0).ToArray();
    }
}

File: js/factorsOfGivenNumber.js
// Create a function that finds each factor of a given number n.
// Your solution should return an array of the number(s) that meet this criteria.
// https://edabit.com/challenge/WR3Q72Jbv7Kyzu6AT

const findFactors = (num) => {
  const factors = [];

  for (let i = 1; i <= num; i++) {
    if (num % i === 0) factors.push(i);
  }

  return factors;
};

module.exports = findFactors;
